packages/react/src/components/Toolbar/Toolbar.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import * as customPropTypes from '@stardust-ui/react-proptypes'
+
+import {
+  childrenExist,
+  createShorthandFactory,
+  UIComponent,
+  UIComponentProps,
+  ContentComponentProps,
+  ChildrenComponentProps,
+  commonPropTypes,
+  ColorComponentProps,
+} from '../../lib'
+import { Accessibility } from '../../lib/accessibility/types'
+import { defaultBehavior } from '../../lib/accessibility'
+import { ShorthandCollection, WithAsProp, withSafeTypeForAs } from '../../types'
+
+import ToolbarItem from './ToolbarItem'
+import ToolbarDivider from './ToolbarDivider'
+import ToolbarRadioGroup from './ToolbarRadioGroup'
+
+export type ToolbarItemShorthandKinds = 'divider' | 'item' | 'group'
+
+export interface ToolbarProps
+  extends UIComponentProps,
+    ContentComponentProps,
+    ChildrenComponentProps,
+    ColorComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   * @default defaultBehavior
+   */
+  accessibility?: Accessibility
+
+  /** Shorthand array of props for Toolbar. */
+  items?: ShorthandCollection<ToolbarItemShorthandKinds>
+}
+
+class Toolbar extends UIComponent<WithAsProp<ToolbarProps>, any> {
+  static create: Function
+
+  static className = 'ui-toolbar'
+
+  static displayName = 'Toolbar'
+
+  static propTypes = {
+    ...commonPropTypes.createCommon(),
+    items: customPropTypes.collectionShorthandWithKindProp(['divider', 'item', 'group']),
+  }
+
+  static defaultProps = {
+    accessibility: defaultBehavior,
+  }
+
+  static Item = ToolbarItem
+  static Divider = ToolbarDivider
+  static RadioGroup = ToolbarRadioGroup
+
+  handleItemOverrides = variables => predefinedProps => ({
+    variables: {
+      ...variables,
```

Comment:
A bit confused why we don't have this in the MenuItem as well...

Reply:
We will - #1392